Automattic/woocommerce-payments
# Fix subscriptions price strings for variations and sign up fees.

Fixes #2122 #2130

#### Changes proposed in this Pull Request

* Added filters for Subscriptions price strings.
* No tests added due to Subs has to be installed for this.

#### Testing instructions

* Use a variable subscription product that has price differences between variations, and also a sign up fee for at least one variation.
* View product, and change currency.
* Confirm that from and sign up fee amounts are converted
* Select a variation and confirm price string below variation selection is converted.

![Screen Capture on 2021-06-15 at 12-18-33](https://user-images.githubusercontent.com/4634416/122079644-fc52a400-cdd3-11eb-8ce4-d09ac4541ee6.gif)


-------------------
